src/ping-thread.coffee

```diff
@@ -0,0 +1,7 @@
+module.exports = (robot) ->
+	robot.hear /@(thread|t)\b/i, (context) ->
+		if context.robot.adapter.fetchThreadParticipants?
+			context.robot.adapter.fetchThreadParticipants context, (list) ->
+				list = list.filter (item) -> item not in ['@thread', '@t']
```

Comment:
That feels weird to me, for one there's quite a lot of threads that start with an `@team` and so that would make `@thread` almost equivalent to `@team` in those cases (special exception being someone in the thread who isn't in the team list) - I feel like it like it should be more a pinging people who have actively participated in the thread.

cc @alexandrosm, what are your thoughts on the `@t` targets?